Versions-mac.plist.in

```diff
@@ -33,6 +33,9 @@
 	<key>Features</key>
 	<array>
 		<string>sgen-concurrent-gc</string>
+		<string>activation</string>
+		<string>modern-http-client</string>
```

Comment:
Sure, I can change that. I just used modern-http-client since the property in MacDev was called SupportsModernHttpClient :)